File: JS/Tarjeta.js
const Payment = require('./Payment');

class Tarjeta extends Payment {
    idCard;
    cvv;
    date;

    constructor(id, idCard, cvv, date) {
        super(id);
        this.idCard = idCard;
        this.cvv =cvv;
        this.date = date;
    }
}

File: Java/Tarjeta.java
class Tarjeta extends Payment {
    Integer idcard;
    Integer cvv;
    Date date;

    public Tarjeta( Integer id, Integer idCard, Integer cvv, Date date ) {
        super( id );
        this.idCard = idCard;
        this.cvv = cvv;
        this.date = date;
    }
}